src/domain/unifiedSearch/useUnifiedSearchQuery.ts

```diff
@@ -64,8 +64,6 @@ export default function useUnifiedSearchQuery<TData = any>(
       ...searchParams,
       ...variables,
     },
-    fetchPolicy: "cache-and-network",
-    nextFetchPolicy: "cache-only",
```

Comment:
@Jokinen  How does this exactly work? I think this had issues with our setFilters implementation.

Reply:
Ok found out that the searh is actually broken without these. No idea why...

Reply:
I figured that the problem is with the `relayStylePagination` helper which doesn't use any keyArgs for chaching by default. So I added `excludeArgs(["after"])` to it and now it seems to work nicely without:

```
fetchPolicy: "cache-and-network",
    nextFetchPolicy: "cache-only",
```

Reply:
I think that `nextFetchPolicy: "cache-only"` was implemented to allow for nicer behaviour when using the back arrow or browser's back button to navigate back from a venue page into the search result list. It's meant to return the previous content as quickly as possible when the user navigates back, so that the page contains all the elements the user previously viewed, and the browser is able to scroll the user to the previous position in the list.

The `fetchPolicy` is likely earlier code and may address a different issue.

Reply:
I think that with this PR it should be still working like that